Simonyi/Components/MTCS/MTDome.rst

```diff
@@ -1,6 +1,115 @@
 .. _Simonyi-Components-MTDome:
 
-#############################
+######
 MTDome
-#############################
+######
+
+MTDome General description
+==========================
+
+The main purpose of the Simonyi Survey Telescope Dome (MTDome) is to protect the telescope from the environment. It must also:
+
+* Provide a clear optical path for observing that can smoothly follow the motions of the telescope as it acquires and tracks fields on the sky.
+* Minimize the effects of stray and scattered exterior light on the light level within the dome interior.
+* Provide controlled ventilation and flushing of the dome interior with external ambient air during night operation to minimize the thermal effects on image quality.
+* Facilitate maintenance of both the telescope and the Dome itself.
+* Provide a controlled thermal environment during the day for preconditioning and testing.
+* Be light tight during the daytime to enable testing and calibration of the telescope’s optical system.
+* Provide for in-situ repair and maintenance of Simonyi Survey Telescope key components.
+As a result of its wide field of view the Simonyi Survey Telescope is very susceptible to stray light and requires a significant amount of optical baffling. The Dome constitutes one of stray light baffles of the overall Simonyi Survey Telescope system. The Dome enclosure is a cylindrical carousel with an aperture "biparting" shutter and light/wind screen function together to provide the stray light protection around the telescope line of sight by actively following the observing light path. The 34 vent openings with light baffling louvers have the conflicting requirements of simultaneously providing ample airflow and substantial scattered light attenuation. The rotation of the Dome is completely independent from the motion of the telescope. Since the Dome operates at non-trivial velocities while the telescope is observing smooth operation is paramount.
+
+MTDome Main Mechanical Components
+=================================
+
+* *Azimuth Drive and Bearing "Bogie" System (ADBS)*: This is the MTDome rotation mechanism. It has five motors spread evenly along the sides of the dome, the motors imprint velocity, acceleration, and jerk values to the MTDome and have power off software limits. The encoder system allows the determination of absolute position and it is able to determine position through the entire range of motion. It has a locking pin system for mechanical interruption of movement.
+* *Aperture shutter*: When fully open, the aperture shutter shall produce an 11.0 (+/-0.05) meter wide clear aperture continuous over the operational elevation range. The shutters have motorized locking mechanisms to restrain them in both the fully open and fully closed positions. The Aperture Shutter also has a locking pin system for mechanical interruption of movement.
```

Comment:
1. After digging around, I found there is no clarity about that value. Since the official value in the documentation is the current one, I will leave it as it is.
2. Done.